test/com/google/enterprise/adaptor/documentum/DocumentumAdaptorTest.java

```diff
@@ -1995,52 +2013,61 @@ private void testDocContent(Date lastCrawled, Date lastModified,
     String content = "<html><body>Hello</body></html>";
     insertDocument(lastModified, path, mimeType, content);
 
-    MockResponse response = getDocContent(ImmutableMap.<String, String>of(),
+    ByteArrayOutputStream boas = new ByteArrayOutputStream();
+    RecordingResponse response = getDocContent(
+        boas,
+        ImmutableMap.<String, String>of(),
         new MockRequest(docIdFromPath(path, DOCUMENT.pad("aaa")),
             lastCrawled));
 
-    assertDocContent(response, lastModified, mimeType, content,
+    assertDocContent(boas, response, lastModified, mimeType, content,
         expectNoContent);
   }
 
-  private void assertDocContent(MockResponse response, Date lastModified,
+
+  @Rule
```

Comment:
This looks to be unused. Don't forget to remove the 2 imports with it.

Reply:
Yes, thx. Removed it.

Reply:
Done.